### DISCUSSION TITLE
Welcome to obsidian-digital-garden Discussions!

### DISCUSSION THREAD

**Discussion by oleeskild** [oleeskild/obsidian-digital-garden#65] [Announcements]
## 👋 Welcome!
  I’m using Discussions as a place to connect with other users of the plugin. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zheneia** [comment]
hi, i'm saezhe!
i'm currently using digital garden to emulate a "conventional" digital garden described by [this article](https://maggieappleton.com/garden-history).

i only selectively publish pages to my site from my obsidian vault, but i want to publish my pages more widely soon.

i was recently working on a [sitemap page](https://erosaezhe.netlify.app/sitemap/) of my site that uses the obsidian dataview plugin to list all of my pages that have the `dg-publish` tag true.
(i also included in the query a check to see if the arbitrary tag `dg-exclude-map` isnt true)

i'm wondering what different philosophies everyone here uses with their sites.

**Comment by paquitxuelo** [comment]
Hello everyone,
My name is Francesc. I'm trying to build a digital garden following your instructions. It is supposed to be able to embed Excalidraw but in the deployment page  it only shows up the code, not the Excalidraw graph. Would you please give some hint? 
Thanks in advance,
Francesc

**Reply by oleeskild** [reply]
Hi 👋
I'll try to help. Do you have a link to the page that is not showing the excalidraw drawing correctly?

**Comment by paquitxuelo** [comment]
Hi Ole,

Here’s the link: https://symphonious-biscuit-e557e2.netlify.app/disseny-basic/

Many thanks,
Francesc

De: Ole Eskild ***@***.***>
Enviat: dilluns, 15 d’agost de 2022 19:01
Per a: ***@***.***>
A/c: ***@***.***>; ***@***.***>
Tema: Re: [oleeskild/obsidian-digital-garden] Welcome to obsidian-digital-garden Discussions! (Discussion #65)


Hi 👋
I'll try to help. Do you have a link to the page that is not showing the excalidraw drawing correctly?

**Comment by oyes77** [comment]
# Hi!
 
### I'm Plank (Tablita in Spanish)
A digital artist and enthusiast in a bunch of stuff, i found an easy way to build a simple webpage here where i can collect info about what i do and my interests to share with my friends and community, in a weird mix of digital garden and blog, i'm not trying to stick to any special order or format on what i publish for now.

I'm still experimenting with themes, i want to find the perfect one that suits my needs regarding image formatting, my digital garden is still in constant change si you may find something different everytime you check it out.
### https://tablondeltablon.netlify.app/   (it's in spanish)

**Comment by hystrix0** [comment]
Hi Ole,

Thanks so much for this contribution. 

Would this set up with Netlify work with a private Github repo and a password secured website? While I am all for public digital gardens, I am thinking of something more like an Obsidian backed shared wiki space for a small team? We already synch Obsidian with Github, but I am thinking this would render our shared content much better than Github does by default with .md files.

**Reply by oleeskild** [reply]
Yes that would work. You would need to publish the md files with the plugin though, as it is doing some processing before uploading it to github. But netlify will work just fine with a private repo.

Netlify also support setting a password for your site, but unfortunately you need a paid plan to do so.

**Comment by tktongas** [comment]
Hi Sir,
I am trying to use this plugin for my digital garden. I already build it at netfily.
During I upload my markdown files, I met some problems:

1. If the markdown file is located in the root path, it can be uploaded and displayed at netfily.
2. if the markdown file is located in the subfolder, such as root/CS/, it can be uploaded to github, while cannot displayed at netfily.

I check the URL at the netfily, the root markdown file has right URL, and the subfolder markdown has wrong URL.

Does it mean, I must put all the markdown files at root path, so they can uploaded and displayed?  

Thanks~

**Reply by oleeskild** [reply]
Hi,
What do you mean by wrong URL? Are you not able to see the note on your site, or do you think the url doesn't match what you expected?

**Reply by tktongas** [reply]
OK,I got it. I renamed my subfolder and the markdown file appeared at netlify.

**Reply by oleeskild** [reply]
Awesome 👍

**Comment by andreiRS** [comment]
Hey guys! Such an amayzing job with the releases 2.22 and 2.23 🚀  I was looking for so long for the navigation tree and search functionality!

Great job to everyone who contributed to these changes! 🎉 

Is there a way to donate for this project! ? would to show my gratitude by at least buying some coffee!

**Reply by oleeskild** [reply]
That really means a lot, thank you for taking the time to write this!

I haven't set up any ways to support the project. But since you asked I've created a [ko-fi page: https://ko-fi.com/oleeskild](https://ko-fi.com/oleeskild) . A coffee would be very appreciated 😄

**Reply by andreiRS** [reply]
Great! Looking forward to future updates!

**Comment by mettaseva** [comment]
@oleeskild ~ 

Love this plugin! Your installation and set-up instructions were clear and easy-to-follow, even for a GitHub/Netlify newbie ~ and the plugin's search feature is _invaluable_! Thank you so much for making this plugin available. 

Also, where would you like us to post feature requests? Under Ideas or Q&A?

Thanks in advance,
Metta

**Comment by BahneGork** [comment]
Great work Ole!
I tried another setup earlier, it sort a worked but i had no idea how to tweak the design/layout, so no sidebar, no graph and frontpage was the infinite blogpost-previews. I like your digital garden alot more. Also i dont have to fiddle with github seperate from obsidian! 

My main use-case for now will be to share player-faced information from my DnD campaign Vault. 
For now i have created a separate "player folder" inside my vault and create notes there. Which atm will mean alot of redundancy until i figure out how to only publish part of notes or put all my GM knowledge inside "%%" comments in my vault, which doesnt seem ideal. But at the same time having 2 notes pr npc or location etc.... also seems not ideal. If anyone else has solved this kind of use-case before with obsidian vault publishing in general im eager to know :D 

all in all very cool plugin !
-michael dk

**Comment by CarolusIII** [comment]
Hello gardeners,
Having been using this plugin for about a month now, I came to the conclusion this is one of the most exciting additions to Obsidian one can think of. For me, it more than doubles the value of this note-making app.
Therefore it is hard to believe why so few people actually share their experiences here.
I am thrilled by the idea I can produce very private or intimate ideas while at the same time sharing parts of it, all in the same app, all from the same pool of thoughts. This is brilliant.
What I miss is community support. I am struggling to implement a handwrite font, to emphasize certain personal thoughts. Is there anybody around that would be willing to help me with just a small CSS question? 
Have a great day.

**Comment by russellgordon** [comment]
Hello all,

I wanted to let Ole, current users of the Digital Garden plugin, and those interested in using the plugin know that I shared a [teaching-centric introduction to publishing with Obsidian and Digital Garden](https://teaching.russellgordon.ca/cemc/sccst-2023/a-rapid-workflow-for-publishing-cs-teaching-materials/) today at a [conference](https://cemc2.math.uwaterloo.ca/events/sinstitute/) at the [University of Waterloo](https://uwaterloo.ca).

[Responses were enthusiastic](https://techhub.social/@siriusnerd/110906759224316070) from the teachers in the room – my gratitude to Ole for authoring and continuing to improve the Digital Garden software.

If you think the information I shared today might be helpful to yourself or others you know, please feel free to share.

Russell Gordon
Lakefield, Ontario, Canada

**Reply by CarolusIII** [reply]
Nice piece of work. Thanks for sharing this.
